nix/hm-module.nix

```diff
@@ -162,6 +162,77 @@ in {
       ];
     };
 
+    pictures = mkOption {
+      description = "Picture configurations";
+      type = listOf (submodule {
+        options = {
+          monitor = mkOption {
+            description = "The monitor to draw an image";
+            type = str;
+            default = "";
+          };
+
+          path = mkOption {
+            description = "The path to source image";
+            type = str;
+            default = "echo '/home/me/cutie.png'"; # only png supported for now
```

Comment:
Why not just 
```suggestion
            default = "/home/me/cutie.png"; # only png supported for now
```

Reply:
i also was surprised, just copy-pasted from background, thought this is intended